src/debugging-support-in-rustc.md

```diff
@@ -104,6 +113,27 @@ LLDB has Rust-like value and type output.
 * LLDB has a plugin architecture but that does not work for language support.
 * GDB generally works better on Linux.
 
+### WinDbg/CDB
+
+Microsoft provides [Windows Debugging Tools] such as the Windows Debugger (WinDbg) and
+the Console Debgugger (CDB) which both support debugging programs written in Rust. These
+debuggers parse the debug info for a binary from the `PDB`, if available, to construct a
+visualization to serve up in the debugger.
+
+#### Natvis
+
+Both WinDbg and CDB support defining and viewing custom visualizations for any given type
+within the debugger using the Natvis framework. The Rust compiler defines a set of Natvis
+files that define custom visualizations for a subset of types in the standard libraries such
+as, `std, core and alloc`. These Natvis files are embedded into `PDBs` generated by the
+`*-pc-windows-msvc` target triples to automatically enable these custom visualizations when
+debugging. This default can be overridden by setting the `strip` rustc flag to either `debuginfo`
+or `symbols`.
+
+Rust has support for embedding Natvis files for crates outside of the standard libraries by
+using the `#[debugger_visualizer]` attribute. For more details on how to embed debugger visualizers
+please refer to the `#[debugger_visualizer]` attribute in [the unstable book](https://doc.rust-lang.org/unstable-book/).
```

Comment:
Maybe it's helpful to link to https://doc.rust-lang.org/unstable-book/language-features/debugger-visualizer.html?

Reply:
Updated. Thanks